slick.core.js

```diff
@@ -760,6 +760,16 @@
     callback();
   }
 
+  function applyDefaults(targetObj, srcObj) {
+    for (const key in srcObj) {
+      if (srcObj.hasOwnProperty(key)) {
+        if (!targetObj.hasOwnProperty(key)) {
```

Comment:
these nested `if` could be regrouped under a single `if` since there's only 1 execution logic underneath

Reply:
true